app/styles/ui/_dropdown-select-button.scss

```diff
@@ -89,45 +89,28 @@
     padding: 0;
     margin: 0;
 
-    :first-child.button-component {
```

Comment:
.... This border-top feels important.. but I didn't see why. :/

Reply:
It was.. but it didn't work. :D  In [ef22059](https://github.com/desktop/desktop/pull/17271/commits/ef22059bca744efc2e92793b227406d52c37d005), I fiddled with the css such that when the dropdown is open there is a visible box color border between button and the blue highlighted menu item so they don't blend together. I used `outline` instead of `border` so that the edges of the dropdown would line up with the edges of the buttons (the slight indentation was annoying me).